### ISSUE TITLE
Error message when adding an additional IP

### ISSUE THREAD

**Issue by tamazlykar** [bwsw/cloudstack-ui#1134]
### Steps to reproduce
1. Select a VM
2. Go to the Network tab
3. Add a secondary IP

### Actual result

An error appears, the rule is added, but it is visible after the page is updated.

### Expected result

No error appears and a rule is visible right after creation.

### Environment
<pre><code>
Browser:
- [ ] Chrome version XX
- [ ] Firefox version XX
- [ ] Safari version XX
- [ ] Opera version XX
- [ ] IE version XX
- [ ] Edge version XX

Others:

</code></pre>

### Connected feature ID

vm_NIC_secondIP_add

### Screenshots

![screenshot from 2018-06-27 11-21-07](https://user-images.githubusercontent.com/18456914/41952931-d9018d46-79fc-11e8-99c7-223e4f2b5efb.png)

**Comment by tamazlykar** [comment]
Another issue appears when you delete secondary IP. IP is deleted but this visible only after page reload.

**Comment by tamazlykar** [comment]
**Test on:**
tamazlykar/1134-additional-ip

**Regression:**
vm_NIC_secondIP_add
vm_NIC_secondIP_remove

**Comment by rennervo** [comment]
Tested on tamazlykar/1134-additional-ip/
